DataConnectors/Perimeter81ActivityLogs.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "Perimeter81ActivityLogs",
+  "title": "Perimeter 81 Activity Logs",
+  "publisher": "Perimeter 81",
+  "descriptionMarkdown": "The Perimeter 81 Activity Logs connector allows you to easily connect your Perimeter 81 activity logs with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation.",
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "Perimeter81_CL",
+      "baseQuery": "Perimeter81_CL"
+    }
+  ],
+  "sampleQueries": [
+    {
```

Comment:
Since these are activity logs, it'll be great to add queries that pivots on list of users with anomalous activities.